src/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  /* margin: 0; */
+  box-sizing: border-box;
+}
+
+.page-content {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+  max-width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  align-self: center;
+
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  margin: 0 0 4px;
+  line-height: 18px;
+}
+
+.card__code {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 0 0 16px;
+  line-height: 14px;
+}
+
+.card__review {
+  display: flex;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 9px;
+}
+
+.stars__star {
+  display: flex;
+  background: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 17px;
```

Comment:
you can add justify-content: space-between for card__review and delete this margin. If your card width change, you will need update this margin too